blog/progress-report-jan-2023/index.mdx

```diff
@@ -92,7 +92,7 @@ If you think you've found a graphical issue, please reach out to us if so!
 Issues we are already currently aware of:
 
 - Red eco rings on the final boss
-  :::
+:::
```

Comment:
happened to notice this small bug in the progress report when grabbing a description about emerc